packages/blade/.storybook/react/preview.js

```diff
@@ -38,12 +39,60 @@ export const parameters = {
   },
   docs: {
     theme,
+    components: {
+      summary: styled.summary`
+        font-family: ${theme.fontBase};
+        color: ${theme.textColor};
+        font-weight: normal;
+        cursor: pointer;
+      `,
+      li: styled.li`
+        :not(:first-child) {
+          padding-top: 16px;
+        }
+        font-size: 14px;
+
+        & :not(pre) > code {
+          line-height: 1;
+          margin: 0 2px;
+          padding: 3px 5px;
+          white-space: nowrap;
+          border-radius: 3px;
+          font-size: 13px;
+          border: 1px solid #eeeeee;
+          background-color: #f8f8f8;
+        }
+      `,
+    },
   },
 };
 
+const StoryCanvas = styled.div(
```

Comment:
curious: where did you find this thing? the docs of storybook itself is not good. finding something known itself is a challenge wondering how did you discover unknown 😄

Reply:
I don't think so it is there in docs. I just saw `BladeProvider` and thought adding a component inside this will give access to themes and theme changes 🙈